.github/workflows/tests.yml

```diff
@@ -80,6 +81,10 @@ jobs:
           sudo microceph.ceph health | grep -xF "HEALTH_OK"
           pgrep ceph-osd || { echo "No ceph-osd process found" ; exit 1; }
 
+      - name: consider debugging
+        uses: lhotari/action-upterm@v1
+        if: failure()
+          
```

Comment:
Do we want to always have that?
In theory that may cause some Github Actions to fail and hit the block for the shell when the user isn't paying attention, causing for long running Github Actions which could end up being costly depending on what plan Canonical is running on these days.